README.md

```diff
@@ -4,24 +4,32 @@
 
 # Sinopia Indexing Pipeline
 
-This is the repository for the Sinopia Indexing Pipeline. The pipeline is a Node application that listens for messages (sent to a queue via STOMP), and for each message (a W3C Activity Streams message):
+This is the repository for the Sinopia Indexing Pipeline. The pipeline is a Node application that
+responds to a MongoDB [Change Streams](https://docs.mongodb.com/manual/changeStreams/) and
```

Comment:
I think the `a` may be unnecessary here since `Change Streams` is plural.

Reply:
Fixed with the latest commit.